_sections/cherry-picking-vscode.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Visual Studio Code
+tab: vscode
+---
+
+### Overview
+
+GitLive's cherry picking feature builds on live diff views, [shown in the visibility section](/docs/visibility){:class="internal-link"}, by allowing developers to cherry pick uncommitted code from their teammates.
+
+Click on your teammate’s working copy and select the cherry icon. You can then enter a commit message and stage the changes that you have just copied for commit on your branch straight away.
```

Comment:
why are you mentioning the possibility about staging. its irrelevant to the feature, they can do whatever they want with it.   take that out of the gif and the text! make the gif like the one for jetbrains